### ISSUE TITLE
Can we remove either gender or title from the contact form?

### ISSUE THREAD

**Issue by IvanaHajz** [idmc-labs/idmc-database-feedback#143]
It seems redundant to me to have male/female and also Mr./Mrs.

**Comment by ghost** [comment]
yes if we speak about gender, then we may need to consider male/female/neutral/etc: http://www.telegraph.co.uk/men/the-filter/a-nearly-complete-glossary-of-gender-identities-for-your-next-ce/

**Comment by luisamg** [comment]
I'd find it more useful to keep only M/F
